package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -183,50 +290,213 @@ class WatsonCrickDist(NucPairDist):
 
     Attributes
     ----------
-    times: numpy.ndarray
-        Simulation times for analysis.
     results.pair_distances: numpy.ndarray
         2D array of Watson-Crick basepair distances. First dimension is
         simulation time, second dimension contains the pair distances for
         each each entry pair in strand1 and strand2.
 
         .. versionadded:: 2.4.0
 
+    results.distances: numpy.ndarray
+        stored in a 2d numpy array with first index selecting the Residue pair, 
+        and the second index selecting the frame number
+    times: numpy.ndarray
+            Simulation times for analysis.
 
     Raises
     ------
-    ValueError
-        If the residues given are not amino acids
+    DeprecationWarning
+        If a lits of :class:`~MDAnalysis.core.groups.Residue` is given for
+        :attr:`strand1` of :attr:`strand2` instead of a
```

Comment:
Just bare role for arguments
```suggestion
        `strand1` of `strand2` instead of a
```

Reply:
I went through and removed the all `attr` used for arguments